osquery/tables/system/darwin/smc_keys.cpp

```diff
@@ -451,8 +457,8 @@ void genTemperature(const std::string &key,
     return;
   }
 
-  int intValue = (val[0] * 256 + val[1]) >> 2;
-  float floatValueCelsius = intValue / 64.0;
+  int intValue = (val[0] * 256 + val[1]);
+  float floatValueCelsius = intValue / 256.0;
```

Comment:
Yep I figured #1825, #1826 would get merged first and then I'll rebase against master.